v23.1/hashicorp-integration.md

```diff
@@ -7,40 +7,52 @@ docs_area: reference.third_party_support
 
 [Vault's Key Management secrets engine](https://www.vaultproject.io/docs/secrets/key-management) allows customers to manage encryption keys on external key management services (KMS) such as those offered by Google Cloud Platform (GCP) or Amazon Web Services (AWS).
 
 CockroachDB customers can integrate these services, using Vault's KMS secrets engine to handle the full lifecycle of the encryption keys that {{ site.data.products.dedicated }} uses to protect their data.
 
-<a id="use-vaults-transit-secrets-engine-to-manage-a-cockroachdb-self-hosted-clusters-encryption-at-rest-store-key"></a>
+Resources:
 
-## Use Vault's Transit Secrets Engine to manage a {{ site.data.products.core }} cluster's {{ site.data.products.enterprise }} Encryption At Rest store key
+- [CMEK overview](../cockroachcloud/cmek.html)
+- [CMEK FAQ](../cockroachcloud/cmek-faq.html)
+- [Manage Customer-Managed Encryption Keys (CMEK) for CockroachDB Dedicated](../cockroachcloud/managing-cmek.html)
+- [Provisioning GCP KMS Keys and Service Accounts for CMEK](../cockroachcloud/cmek-ops-gcp.html)
+- [Provisioning AWS KMS Keys and IAM Roles for CMEK](../cockroachcloud/cmek-ops-aws.html)
 
-When deploying {{ site.data.products.enterprise }}, customers can provide their own externally managed encryption keys for use as the *store key* for CockroachDB's [{{ site.data.products.enterprise }} Encryption At Rest](security-reference/encryption.html#encryption-at-rest-enterprise).
+## Use Vault's PKI Secrets Engine to manage a {{ site.data.products.dedicated }} cluster's certificate authority (CA) and client certificates.
 
-Vault's [Transit Secrets Engine](https://www.vaultproject.io/docs/secrets/transit) can be used to generate suitable encryption keys for use as your cluster's store key.
+{{ site.data.products.dedicated }} customers can use Vault's public key infrastructure (PKI) secrets engine to manage PKI certificates for client authentication to the cluster. Vault's PKI Secrets Engine greatly eases the security-critical work involved in maintaining a certificate authority (CA), generating, signing and distributing PKI certificates.
 
-<a id="use-vaults-pki-secrets-engine-to-manage-a-cockroachdb-self-hosted-clusters-server-and-client-certificates"></a>
+By using Vault to manage certificates, you can use only certificates with short validity durations, an important component of PKI security.
+
+Refer to [Transport Layer Security (TLS) and Public Key Infrastructure (PKI)](security-reference/transport-layer-security.html) for an overview.
 
-## Use Vault's PKI Secrets Engine to manage a {{ site.data.products.core }} cluster's server and client certificates
+Refer to [CockroachDB Cloud: Certificate Authentication for SQL Clients](../cockroachcloud/client-certs-dedicated.html) for procedures in involved in adminstering PKI for a {{ site.data.products.dedicated }} cluster.
```

Comment:
I don't think this needs CockroachDB Cloud in the link text. WDYT?

Reply:
I disagree, and am changing the title back to having Cloud in the title (I removed it based on your suggestion on the client certs PR. It needs it to disambiguate from the self-hosted PKI page.